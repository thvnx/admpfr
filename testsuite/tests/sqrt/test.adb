with Ada.Text_IO; use Ada.Text_IO;
with Admpfr;      use Admpfr;

procedure Test is
   M, N : Mpfloat;
begin
   N.Set (Float (1.5));

   M.Sqrt (N);
   Put_Line (M'Image);

   M.Sqrt (4);
   Put_Line (M'Image);

   M.Rec_Sqrt (N);
   Put_Line (M'Image);
end Test;
