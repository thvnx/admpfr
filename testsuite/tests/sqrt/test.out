1.2247448713915889e+00
2.0000000000000000e+00
8.1649658092772603e-01
